packages/perseus/src/widgets/interactive-graphs/legacy-grid.tsx

```diff
@@ -0,0 +1,49 @@
+import {View} from "@khanacademy/wonder-blocks-core";
+import * as React from "react";
+
+import AssetContext from "../../asset-context";
+import {SvgImage} from "../../components";
+import {interactiveSizes} from "../../styles/constants";
+import {getInteractiveBoxFromSizeClass} from "../../util/sizing-utils";
+
+import type {PerseusImageBackground} from "../../perseus-types";
+import type {SizeClass} from "../../util/sizing-utils";
+
+/**
+ * If a graphie-to-png URL is provided in `backgroundImage`, will return the rendered graphie background.
+ * Otherwise, returns `null`.
+ */
+export const getLegacyGrid = (
+    containerSizeClass: SizeClass,
+    backgroundImage?: PerseusImageBackground,
+) => {
+    const {url, width, height} = backgroundImage ?? {};
+    if (url && typeof url === "string") {
+        const box = getInteractiveBoxFromSizeClass(containerSizeClass);
+        const scale = box[0] / interactiveSizes.defaultBoxSize;
+        return (
+            <View
+                style={{
+                    position: "absolute",
+                    bottom: 0,
+                    left: 0,
```

Comment:
**question**: Is the bottom, left anchoring due to the fact that we put the `y` label at the top of the graph and the `x` to the right?

Reply:
Yes!